lib/asset_cloud/buckets/gcs_bucket.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+gem "google-cloud-storage", "~> 1.11"
```

Comment:
I haven't seen this before. I assume it enforces that if this file is required there must be a compatible version of the gem available, before requiring it?

Reply:
https://github.com/rails/rails/blob/e1a9b3d861cd6d893541a4e6e14fb68f6891e846/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L6

This is how Rails loads peer dependencies. Other gems do it as well. It works as long as you're using bundler.